### DISCUSSION TITLE
Where is the archive button

### DISCUSSION THREAD

**Discussion by rikki78** [photoprism/photoprism#2315] [Q&A]
I've read there is an archive button in photoprism, to clean up unwanted photos. But I can't find it... I've activated the Delete button checkbox in the settings already but this didn't help. Where can I find this function?

**Comment by graciousgrey** [comment]
It's in the context menu

https://docs.photoprism.app/user-guide/organize/archive/

Delete is only available from within the archive:

https://docs.photoprism.app/user-guide/organize/delete/

**Reply by rikki78** [reply]
Sorry, my question wasn't clear. 
I meant to ask, where is the archive? I found the archive button but the archive itself doesn't show up.

**Reply by rickysarraf** [reply]
This one
![Screenshot_20220510_230849](https://user-images.githubusercontent.com/65480/167689435-6707d46a-24e2-4fbb-b6e7-2bb0856f3577.jpeg)

**Reply by rikki78** [reply]
Exactly, that one. I overlooked the dropdown on the Search, completely
